.github/workflows/CI.yml

```diff
@@ -942,6 +942,7 @@ jobs:
             git
             mingw-w64-ucrt-x86_64-boost
             mingw-w64-ucrt-x86_64-cmake
+            mingw-w64-ucrt-x86_64-cppwinrt
```

Comment:
Please add this to the build docs and https://github.com/LizardByte/Sunshine/blob/7702e55476fef7edf7e421a3631dcde2686afe73/.codeql-prebuild-cpp-Windows.sh#L8

Reply:
I must have forgotten to make changes to the codeql file in #2323; I added them here.